assets/css/gutenberg-syndicated-post.scss

```diff
@@ -40,6 +47,7 @@ body.dt-linked-post {
 
 	.components-panel__body-toggle,
 	.edit-post-post-visibility__toggle,
+	.editor-post-switch-to-draft,
```

Comment:
@av3nger As the main issue in #990 was allowing users to move a distributed copy of a post back to draft, I've pushed this to your PR to account for UI changes in WP 6.3.